lib/cisco_node_utils/cmd_ref/pim.yaml

```diff
@@ -0,0 +1,20 @@
+# pim
+---
+_template:
+  config_get : "show run pim all"
+  config_get_token :
+    - '/^vrf context <vrf>$/'
+  config_set :
+    - "vrf context <vrf>"
+
+all_ssm_ranges:
+  multiple:
+  config_get_token_append : '/^\s*ip pim ssm range (.*)$/'
```

Comment:
`'^ip pim ssm range (.*)$/'

The command_ref code will strip out all leading spaces.